automation/test-linux.sh

```diff
@@ -108,7 +139,13 @@ run_vm(){
     fi
     set -e
   
-    delete_vm $vm_name $template_name
+    if [ "${KUBE_CMD}" == "oc" ]; then
```

Comment:
Can you please create new variable, which will hold these `oc` and `kubectl` constants? So if something change, you will not have to change it on every line.

Reply:
Done